account-keys/test-vectors/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "mc-account-keys-test-vectors"
+version = "0.5.0"
+authors = ["MobileCoin"]
+edition = "2018"
+
+[dependencies]
+mc-account-keys-test-vectors-structs = { path = "structs" }
```

Comment:
Why do you need both `mc-account-keys-test-vectors-structs` and `mc-account-keys-test-vectors`?

Reply:
To my knowledge, build scripts are the only way of generating a file during compilation (a `bin` could be used, but then it makes testing a multi-step process). Because the build script cannot depend on itself and can't use code in `lib.rs`, the code either has to reside in the `build.rs` itself or in a separate crate. I wanted the structs that are used for generating the jsonl files to also be used in deserializing those same files, so they can't reside in the `build.rs` (or at least not without even more code gen). This means the structs either need to live in `mc-account-keys` itself or in a completely separate crate.

If you have any suggestions, I would love to not have 2 extra crates. I couldn't figure out a better way though.